src/commands/login.ts

```diff
@@ -0,0 +1,69 @@
+import {AuthenticationClient} from 'auth0';
+import {Command, flags} from '@oclif/command';
+import chalk from 'chalk';
+import * as inquirer from 'inquirer';
+import * as keytar from 'keytar';
+
+const _error = chalk.red;
+const _success = chalk.green;
+
+export default class Login extends Command {
+  static description = 'Log in to a Architect registry';
+
+  static flags = {
+    help: flags.help({char: 'h'}),
+    username: flags.string({
+      char: 'u',
+      description: 'Username'
+    }),
+    password: flags.string({
+      char: 'p',
+      description: 'Password'
+    })
+  };
+
+  async run() {
+    this.log('Login with your Architect ID to push and pull images from Architect Hub.')
+    const answers: any = await this.promptOptions();
+    try {
+      await this.login(answers.username, answers.password);
+    } catch (err) {
+      this.error(_error(err.message));
+    }
+  }
+
+  async promptOptions() {
+    const {flags} = this.parse(Login);
+    return inquirer.prompt([{
```

Comment:
Let's skip the prompt if the flags are passed in during command execution. That would help logins on headless flows like CI pipelines.